# message a robot sends to the task planner when it started a task from its queue

bool started
uint32 taskId